shared/errors.go

```diff
@@ -0,0 +1,17 @@
+package shared
```

Comment:
What's the difference between `common` and `shared`?

Reply:
I removed common layer.